dnfdragora/ui.py

```diff
@@ -1072,9 +1073,9 @@ def _setInfoOnWidget(self, pkg) :
                     if pkg.updateinfo :
                         s += '<b>%s</b>'%pkg.updateinfo[0]['title']
```

Comment:
Since we don't know what title contains i'd escape that too :)

Reply:
OK